test/functional/eval/timer_spec.lua

```diff
@@ -20,8 +20,10 @@ describe('timers', function()
 
   it('works one-shot', function()
     eq(0,eval("[timer_start(10, 'MyHandler'), g:val][1]"))
-    run(nil, nil, nil, load_adjust(50))
-    eq(1,eval("g:val"))
+    run(nil, nil, nil, load_adjust(10))
+    retry(nil, load_adjust(200), function()
+      eq(1,eval("g:val"))
```

Comment:
@justinmk 
How do you feel about adjusting the code style (at least for changed/touched lines), i.e. add a space after "eq(1," - or is that actually preferred?

Reply:
sure, that's what we usually do.